### DISCUSSION TITLE
Is there any way of forcing dark mode?

### DISCUSSION THREAD

**Discussion by KonVei82** [wg-easy/wg-easy#799] [Q&A]
Dark mode is great but having it automatically apply based on global device settings is awkward. What if i want only the UI web page to be in dark mode but the rest of the browser not? What if i am not in front of my PC? There should be a toggle or at least a three way ENV UI_DARK_MODE=on/off/auto.
Or maybe i am missing something and it is possible to force without forcing your whole browser into a dark mode.

**Comment by sym0nd0** [comment]
I've been trying to find this option, but I've drawn a blank.

My laptop is always in dark mode but the web gui for wg-easy is still in light. 😕 I also don't seem to have the activity graphs (I assume up and down speeds). Thought I'd followed the guides to the letter but thinking I've missed something.

**Reply by suxscribe** [reply]
Until there's a control for dark mode It's possible to do with extension like [Dark Reader](https://darkreader.org). It allows enable dark mode on specific site only

**Comment by peterlewis** [comment]
Having spoken with @WeeJeWel about this during project maintainer handover, we understand that he is keen to avoid visual clutter. For that reason we opted for an automatic switch to align with the OS theme.

However, if you're happy to put together a PR that makes use of an optional variable that can be set to `dark` or `light` (for which preferably the default `auto` would then respect OS theme) then I'm sure we'd be delighted to merge it!

There's a tailwind guide that steps through this process at:

https://tailwindcss.com/docs/dark-mode#supporting-system-preference-and-manual-selection

I've checked and we didn't remove the logic from the previous dark mode toggle (instead simply opting to remove from source and manually set to `media`), so there's a starter for ten at:

https://github.com/wg-easy/wg-easy/blob/cebc5de57922dfa5941e463175db101728457ad1/src/www/js/app.js#L251C1-L261C5

Happy to assist if you get stuck! Thanks! 👍🏻

**Comment by KonVei82** [comment]
Thank you for the dark mode toggle in latest version. Appreciate it.

**Comment by cezaroscan** [comment]
If you want the default startup theme to be dark, a very easy way is to modify line 101 in app.js from:

uiTheme: localStorage.theme || 'auto',

..to 

uiTheme: localStorage.theme || 'dark',
